gitlabbot/gitlabbot/http.go

```diff
@@ -120,12 +121,49 @@ func (h *HTTPSrv) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	shouldNotifyConvs := len(convs) == 0
```

Comment:
we should always check for notifications, we may have something to notify but not everything